sdk/messaging/eventgrid/azsystemevents/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 0.6.2 (2025-05-06)
+## 0.6.2 (TBD)
```

Comment:
Looks like there are breaking changes so this should be `0.7.0`.

Reply:
Good catch. Looks like that field rename should NOT have happened, so I'm fixing the typespec.

Reply:
Updated. Also, added in an entry for the breaking change.